src/result_object/index.js

```diff
@@ -196,6 +196,34 @@ class ResultObject {
         return [];
     }
 
+    /**
+     * Returns whether a field (or the whole suite, if none passed) contains errors
+     * @param {string} [fieldName]
+     */
+    hasErrors(fieldName: string): boolean {
+        if (!fieldName) {
+            return this.hasValidationErrors;
+        }
+
+        return this.validationErrors[fieldName]
```

Comment:
<img width="1168" alt="screen shot 2019-01-05 at 22 47 38" src="https://user-images.githubusercontent.com/7091543/50729062-f0128900-113b-11e9-8df3-2e63bc823aa1.png">

Why not using already implemented getErrors().length?

Reply:
That's actually a good idea. 👍